content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -225,66 +291,77 @@ Admin URL: https://app.netlify.com/sites/YOUR_SITE_NAME
-The URL should like this: `https://YOUR_SITE_NAME.netlify.app`.
+### Result
+
+Your default browser opens the static frontend that you deployed.
+The URL should look like this: `https://YOUR_SITE_NAME.netlify.app`.
 
-![deployed-screenshot](https://res.cloudinary.com/prismaio/image/upload/v1628765208/docs/RnaHMS3_urf3iu.png)
+![Netlify application with Prisma deployed](images/500-08-deploying-to-netlify-application-deployed.png)
 
 The four buttons allow you to make requests to the REST API and view the response:
 
-- **Check API status**: Will call the REST API status endpoint that returns `{"up":true}`. The implementation code is in `functions/index.js`
-- **Seed data**: Will delete all database records and load the database with test data `users`, `profiles`, and `posts`. Returns the created users. The implementation code is in `functions/seed.js`
-- **Load users with profiles**: Will load all `users` in the database with their related `profiles`. The implementation code is in `functions/getUsers.js`
-- **Load Posts**: Will load `posts` and their related `authors`. The implementation code is in `functions/getPosts.js`
+- **Check API status**. Calls the REST API status endpoint that returns `{"up":true}`. The implementation code is in `functions/index.js`
+- **Seed data**. Deletes all database records and loads the database with test data `users`, `profiles`, and `posts`. Returns the created users. The implementation code is in `functions/seed.js`
+- **Load users with profiles**: Loads all `users` in the database with their related `profiles`. The implementation code is in `functions/getUsers.js`
+- **Load Posts**: Loads `posts` and their related `authors`. The implementation code is in `functions/getPosts.js`
 
-For example, calling seed data should show the following:
+For example, when you slick Seed data, the application shows the following:
 
-![deployed-screenshot](https://res.cloudinary.com/prismaio/image/upload/v1628765199/docs/dOJUBsv_e6fx9m.png)
+![Netlify application with Prisma deployed - Call result](images/500-09-deploying-to-netlify-application-deployed-call-result.png)
 
 ## Summary
 
-Congratulations! You have successfully deployed the application to Netlify.
+You have successfully deployed an application that uses Prisma to Netlify.
+
+For more insight into Prisma Client API, you can explore the function handlers in the `functions/` folder.
+
+When you use a FaaS (function as a service) environment to interact with a database, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to the database (this is not a problem with a constantly running node.js server).
 
-For more insight into Prisma Client's API, look at the function handlers in the `functions/` folder.
+You can use the [Data Proxy](/data-platform/data-proxy) for connection pooling and to reduce your Prisma Client bundle and avoid cold starts.
```

Comment:
I wasn't quite sure what this meant, so I've suggested two potential changes. Suggestion 1:

```suggestion
You can use the [Data Proxy](/data-platform/data-proxy) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
```

Reply:
Suggestion 2

However, if the last two items are part of the same logical unit (because avoiding cold starts is a results of bundle size reduction) then consider recasting to something like:

```suggestion
You can use the [Data Proxy](/data-platform/data-proxy) to reduce your Prisma Client bundle size and avoid cold starts, and for connection pooling.
```